keyboards/handwired/lagrange/transport.c

```diff
@@ -0,0 +1,167 @@
+/* Copyright 2020 Dimitris Papavasiliou <dpapavas@protonmail.ch>
```

Comment:
Is this actually using SPI data? 

If so, then it may be best to use `spi_master.c` rather than a custom implementation.   (located in /drivers/avr/spi_master.c)

Reply:
SPI is used for the communication between the two halves, yes.  I avoided using the existing SPI master driver for two reasons:

1. The existing SPI master driver is written for half-duplex communication.  I transfer data both ways (to update the slave side with the state needed to drive it's LEDs) and data is exchanged in parallel.  Using the SPI master driver, I would have to do it sequentially, wasting some time needlessly.
2. There's no matching SPI client driver so that part would need to be low-level AVR code anyway and symmetry would be broken. 

I considered adding the needed functionality to QMK and discussed this on discord.  I believe you were in favor, but I was ultimately advised against it.

Reply:
Ah, okay, no worries, then.